lit_gpt/config.py

```diff
@@ -1037,4 +1037,31 @@ def norm_class(self) -> Type:
         configs.append(copy)
 
 
+###################
+# StatNLP Research
+###################
+
+tiny_LLaMA = [
+    dict(
+        org="StatNLP-research",
+        name="tiny-llama-1b",
```

Comment:
Using a generic model name won't easily work for our tests (see current failure). I would go with the safer option of listing the separate checkpoint configurations separately.

Reply:
I find it unpractical that the names in our config are tied to HF. At least for pretraining it makes not a lot of sense.